commit 314c64ca73a1a0cf79ca6fe55f7aac5f7a7eacb5 (refs/changes/86/1486/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-09T16:43:34-08:00 (2 years, 8 months ago)
    
    TOOLS-1691 'sdcadm post-setup dev-headnode-prov' doesn't refresh the correct config-agent

diff --git a/CHANGES.md b/CHANGES.md
index b55cc20..949033d 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,11 @@
 
 # sdcadm Changelog
 
+## 1.15.5
+
+- TOOLS-1689 Fix a problem where `sdcadm` would hang on exit for many commands that
+  used Ur.
+
 ## 1.15.4
 
 - TOOLS-1683 'sdcadm experimental update AGENT' should support updates of
diff --git a/lib/post-setup/dev-headnode-prov.js b/lib/post-setup/dev-headnode-prov.js
index ed6c8a6..64cc6b5 100644
--- a/lib/post-setup/dev-headnode-prov.js
+++ b/lib/post-setup/dev-headnode-prov.js
@@ -53,7 +53,7 @@ function makeHeadnodeProvisionable(opts, cb) {
             }
 
             progress('Configuring CNAPI to allow headnode provisioning' +
-                ' and over-provisioning (allow a minute to propagate)');
+                ' and over-provisioning');
             var update = {
                 metadata: {
                     ALLOC_FILTER_HEADNODE: false,
@@ -90,6 +90,7 @@ function makeHeadnodeProvisionable(opts, cb) {
                     progress('Refreshing instance %s config-agent', inst.alias);
                     svcadm.svcadmRefresh({
                         server_uuid: inst.server,
+                        zone: inst.instance,
                         wait: true,
                         fmri: 'config-agent',
                         sdcadm: sdcadm,
diff --git a/package.json b/package.json
index bfa6be5..0d0a4ec 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcadm",
   "description": "Administer a SmartDataCenter (SDC) standup",
-  "version": "1.15.4",
+  "version": "1.15.5",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
